src/lib/react/native/component/Image.hx

```diff
@@ -3,6 +3,8 @@ package react.native.component;
 #if react_native
 
 @:jsRequire('react-native', 'Image')
-extern class Image{}
+extern class Image{
+	public static function getSize(uri:String, success:Int->Int->Void , failure:Dynamic->Void):Void;
```

Comment:
failure should be optional

See: https://github.com/facebook/react-native/blob/7263c34/Libraries/Image/Image.android.js#L188